ansible_ai_connect/ai/api/model_pipelines/bam/configuration.py

```diff
@@ -0,0 +1,78 @@
+#  Copyright Red Hat
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from dataclasses import dataclass
+from typing import Optional
+
+from rest_framework import serializers
+
+from ansible_ai_connect.ai.api.model_pipelines.langchain.configuration import (
+    LangchainBasePipelineConfiguration,
+    LangchainConfiguration,
+    LangchainConfigurationSerializer,
+)
+from ansible_ai_connect.ai.api.model_pipelines.registry import Register
+
+# -- Base
+# ANSIBLE_AI_MODEL_MESH_API_URL
+# ANSIBLE_AI_MODEL_MESH_MODEL_ID
+# ANSIBLE_AI_MODEL_MESH_API_TIMEOUT
+
+# -- BAM
+# ANSIBLE_AI_MODEL_MESH_API_KEY
+# ANSIBLE_AI_MODEL_MESH_API_VERIFY_SSL
+
+
+@dataclass
+class BAMConfiguration(LangchainConfiguration):
```

Comment:
For the record, BAM is not a thing anymore and we can remove it.

Reply:
See https://issues.redhat.com/browse/AAP-35807